hoth_api.yaml

```diff
@@ -316,8 +316,17 @@ components:
           type: object
           additionalProperties: true
     GenerateDocumentRequest:
-      oneOf:
-        - $ref: "#/components/schemas/DescriptionOfWork"
+      description: "Determines which document template the API will generate"
+      type: object
+      properties:
+        documentType:
+          type: string
+          enum:
+            - DESCRIPTION_OF_WORK
+        templatePayload:
```

Comment:
Not a fan of the inconsistency here with `documentType` vs `award_history` (camelCase vs snake_case). It seems like a lot of implementation has already been written though... :/

Reply:
Yeah, that was on me. I can open up a separate PR to update both the hoth api and implementation for this to stay consistent with the `snake_case`.